### PR TITLE
virtual host settings and reporting service config changes

### PR DESCRIPTION
**Pull request by sri-tscube**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,7 +104,8 @@ Installing certificates, and configuring the docker overlay:
 4. Make certs:
     1. `cd ~/.dinghy/certs`
     2. `mkcert -cert-file app.lara.docker.crt -key-file app.lara.docker.key app.lara.docker`
-    3. `mkcert -cert-file app.portal.docker.crt -key-file app.portal.docker.key app.portal.docker`
+       [For automation `mkcert -cert-file authoring.dev.docker.crt -key-file authoring.dev.docker.key authoring.dev.docker`]
+    3. `mkcert -cert-file learn.dev.docker.crt -key-file learn.dev.docker.key learn.dev.docker`
```
Comment (scytacki) [inline README.md:108]:
```suggestion
    3. `mkcert -cert-file app.portal.docker.crt -key-file app.portal.docker.key app.portal.docker` 
    4. For automation:
        1.  `mkcert -cert-file authoring.dev.docker.crt -key-file authoring.dev.docker.key authoring.dev.docker`
        2.  `mkcert -cert-file learn.dev.docker.crt -key-file learn.dev.docker.key learn.dev.docker`
```

For a developer following the regular readme setup for lara and the portal the'll still want a cert for `app.portal.docker`

**Review by scytacki — CHANGES_REQUESTED** [review]
I found a few possible changes to make this easier for new developers to setup.

File: README.md [hunk 2]
```diff
@@ -57,6 +57,18 @@ Allowed URL Redirects: 'https://app.lara.docker/users/auth/cc_portal_localhost/c
 2. In the Portal, edit `.env` set SITE_URL=https://app.portal.docker (Ensure that its using https. Othewise there will be communication issues between Portal and lara)
 3. You may need to use the rails console in LARA to set the `is_admin` flag to the portal admin user.
 
+### Communication with reporting service
+To communicate with reporting service you need to make sure you update '.env' with below variables
+1. REPORT_SERVICE_URL=https://us-central1-report-service-dev.cloudfunctions.net/api
+2. REPORT_SERVICE_TOKEN as 'devtoken' for development
+3. REPORT_SERVICE_SELF_URL=https://app.lara.docker (for automation https://authoring.dev.docker) 
+4. REPORT_SERVICE_TOOL_ID with lara host name. Example: app.lara.docker  and for automation authoring.dev.docker
```
Comment (scytacki) [inline README.md:65]: This section should be updated now that the variables the developer needs to set have changed.

File: README.md [hunk 3]
```diff
@@ -92,7 +104,10 @@ Installing certificates, and configuring the docker overlay:
 4. Make certs:
     1. `cd ~/.dinghy/certs`
     2. `mkcert -cert-file app.lara.docker.crt -key-file app.lara.docker.key app.lara.docker`
-    3. `mkcert -cert-file app.portal.docker.crt -key-file app.portal.docker.key app.portal.docker`
+    3. `mkcert -cert-file learn.dev.docker.crt -key-file learn.dev.docker.key learn.dev.docker`
```
Comment (scytacki) [inline README.md:107]:
```suggestion
    3. `mkcert -cert-file app.portal.docker.crt -key-file app.portal.docker.key app.portal.docker`
```
A developer will still want to use `app.portal.docker` for the domain.

**Review by scytacki — CHANGES_REQUESTED** [review]
Changes look good overall, I just found a couple of issues.

File: README.md (hunk 2 above)
Comment (sri-tscube) [reply README.md]: Sorry, my bad. Fixed now. I also removed the token. I hope it's fine.

File: README.md (hunk 3 above)
Comment (sri-tscube) [reply README.md]: Fixed it.

**Review by scytacki — APPROVED** [review]
Looks good.

**Pull request merged**